### PR TITLE
#5482 [Manager] Fail to update metrics when there are stopped shabond…

### PR DESCRIPTION
**Pull request by konekoya**
…is or streams exist in a pipeline

### Checklist

- [x] [Review Contribution Guidelines](https://oharastream.github.io/en/docs/master/contributing/)
- [x] Add at least one reviewer (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add an assignee (of course, you are the assignee by default)
- [x] Do not set milestone, projects and labels (PR labels are added automatically by Jenkins)
- [x] Link to an issue
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (If the failed tests are unrelated to this PR, add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/Pipeline/Paper/Paper.js [hunk 1]
```diff
@@ -838,13 +838,11 @@ const Paper = React.forwardRef((props, ref) => {
...
         objects
-          .filter((object) => object.kind !== KIND.topic) // topic metrics are not displayed in the current UI
-          .forEach(({ nodeMetrics, name }) => {
-            // Since there could be more than one node's metrics data available
-            // in the element, we will only display the first node
-            const firstNode = Object.keys(nodeMetrics)[0];
-            findElementView(name).updateMeters(nodeMetrics[firstNode]);
-          });
+          .filter(({ kind }) => kind !== KIND.topic) // topic metrics are not displayed in the current UI
+          .filter(({ state }) => state?.toLowerCase() === CELL_STATUS.running)
```
Comment (konekoya) [inline ohara-manager/client/src/components/Pipeline/Paper/Paper.js:842]: Only displaying metrics for running elements

File: ohara-manager/client/src/components/Pipeline/Paper/Paper.js [hunk 2]
```diff
@@ -838,13 +838,11 @@ const Paper = React.forwardRef((props, ref) => {
       },
       updateMetrics(objects) {
         objects
-          .filter((object) => object.kind !== KIND.topic) // topic metrics are not displayed in the current UI
-          .forEach(({ nodeMetrics, name }) => {
-            // Since there could be more than one node's metrics data available
-            // in the element, we will only display the first node
-            const firstNode = Object.keys(nodeMetrics)[0];
```
Comment (konekoya) [inline ohara-manager/client/src/components/Pipeline/Paper/Paper.js:845]: We need to factor in multiple nodes scenario

File: ohara-manager/client/src/components/Pipeline/Paper/cell/createConnectorCell.js [hunk 3]
```diff
@@ -349,30 +354,44 @@ function getIcon(kind) {
...
+    const defaultValue = metricsName === defaultText ? '' : 0;
+    const metricsValue = results?.[0]?.value || defaultValue;
+
+    return `
+      <span class="metrics-name">${metricsName}</span>
+      <span class="metrics-value">${metricsValue.toLocaleString()}</span>
+    `;
+  } else if (meters.length === 1) {
+    // Since there's only one item available, display it even if its value is zero
+    const value = meters[0]?.value || 0;
```
Comment (konekoya) [inline ohara-manager/client/src/components/Pipeline/Paper/cell/createConnectorCell.js:383]: This ensure that at least the metrics item is displayed even when it has the value of `0`

**Comment by konekoya** [comment]
We couldn't test this PR as "metrics" can only be tested in real env (e2e tests) but we currently only use one node to run our tests and this bug can only repro with two or more nodes :(

**Comment by saivirtue** [comment]
> We couldn't test this PR as "metrics" can only be tested in real env (e2e tests) but we currently only use one node to run our tests and this bug can only repro with two or more nodes :(

Looks like we should support multiple nodes env for our tests in the next milestone?

**Comment by konekoya** [comment]
> Looks like we should support multiple nodes env for our tests in the next milestone?

Yep, this seems inevitable since our end users need to deploy our App across many nodes...

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
Fail it tests will be addressed in #5480 and has nothing to do with this PR

**Pull request merged**